<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NearestNeighbourSearch</class>
 <widget class="QDialog" name="NearestNeighbourSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter the Query Point</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQP">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>The Closest cab is Present at </string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_res">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_dist">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>having</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>units distance from you.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_pic</zorder>
  <zorder>label</zorder>
  <zorder>lineEditQP</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_2</zorder>
  <zorder>textBrowser_res</zorder>
  <zorder>label_3</zorder>
  <zorder>textBrowser_dist</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
